### DISCUSSION TITLE
global language?

### DISCUSSION THREAD

**Discussion by Huber1** [TandoorRecipes/recipes#1828] [Q&A] [answered]
Hello Everyone,
today i was trying to set the global language of my instance to german, so my family doesn't have to deal with the page in english.
Did I overlook the setting or doesn't it exist?

Thanks in advance,
Moritz

**Comment by vabene1111** [comment, marked as answer]
this is actually missing, i had too look into the code myself because I did not think this was possible. That said django should automaticall detect your language based on your browser settings, is that not working for you ?

**Reply by Huber1** [reply]
Thanks, i didn't think of that. Personally i like English as my Display language but since my Family also uses Tandoor, i thought of a global language. I tried on another device with another display language and it worked correctly, so everyone sees their preferred one
